### PR TITLE
feat: introduce stressnet-wl1

### PR DESCRIPTION
**Pull request by mrekucci**
## Describe your changes

Introduces new stressnet profile with L1 holesky.

## Checklist before requesting a review

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2 [hunk 1]
```diff
@@ -179,6 +180,8 @@ job "{{ job.name }}" {
 
           forge clean --root "${CONTRACT_REPO_ROOT_PATH}"
 
+            {%- if profile != 'stressnet-wl1' %}
```
Comment (aloknerurkar) [inline infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2:183]: nit: fix indent

File: infrastructure/nomad/playbooks/templates/jobs/mev-commit-funder.nomad.j2 [hunk 2]
```diff
@@ -34,12 +34,21 @@ job "{{ job.name }}" {
 
       template {
         data = <<-EOH
-          {%- raw %}
+            {%- raw %}
```
Comment (aloknerurkar) [inline infrastructure/nomad/playbooks/templates/jobs/mev-commit-funder.nomad.j2:37]: These tags dont need to be indented?

**Review by aloknerurkar — APPROVED** [review]

File: infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2 (hunk 1 above)
Comment (mrekucci) [reply infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2:183]: Done.

File: infrastructure/nomad/playbooks/templates/jobs/mev-commit-funder.nomad.j2 (hunk 2 above)
Comment (mrekucci) [reply infrastructure/nomad/playbooks/templates/jobs/mev-commit-funder.nomad.j2:37]: This helps readability.

**Pull request merged**